packages/squiggle-lang/src/value/vLambda.ts

```diff
@@ -53,4 +55,6 @@ export class VLambda extends BaseValue<"Lambda", number> implements Indexable {
   // deserialization is implemented in ./serialize.ts, because of circular import issues.
 }
 
-export const vLambda = (v: Lambda) => new VLambda(v);
+export function vLambda(v: Lambda) {
```

Comment:
Btw, this is an interesting case where `function vNambda` instead of arrow function helped with recursive import issues.